k8s/ace-rp/Makefile

```diff
@@ -20,7 +20,7 @@ OS 				= $(shell uname -s | tr '[:upper:]' '[:lower:]')
 ARCH  				= $(shell uname -m | sed 's/x86_64/amd64/')
 
 #IMAGES
-ACE_RP_IMG				?= ghcr.io/trustbloc-cicd/sandbox-ace-rp:0.1.7-snapshot-50dd889
+ACE_RP_IMG				?= ghcr.io/trustbloc/sandbox-ace-rp:latest
```

Comment:
This will break the downstream stream systems.

These variables are already set in https://github.com/trustbloc/sandbox/blob/main/k8s/Makefile#L11-L15, when we run `make build-setup-deploy`

Can we update PR with just code changes (adding missing contexts) and verify ?

Reply:
https://github.com/trustbloc/sandbox/pull/1075

Reply:
@rolsonquadras fixed